components/monitoring/README.md

```diff
@@ -40,160 +42,138 @@ for adding the service monitor declaration:
-     ...
-       - spi/
-     ...
-  ```
-  - Note: to keep the `ref={id}` up to date such a configuration of PipelineRun can be used
-    ```yaml
-    apiVersion: tekton.dev/v1beta1
-    kind: PipelineRun
+    apiVersion: integreatly.org/v1alpha1
+    kind: GrafanaDashboard
     metadata:
-      name: spi-controller-on-push
-    annotations:
-      pipelinesascode.tekton.dev/on-event: "[push]"
-      pipelinesascode.tekton.dev/on-target-branch: "[main]"
-      pipelinesascode.tekton.dev/max-keep-runs: "5"
+      name: grafana-dashboard-o11y
+      labels:
+        app: appstudio-grafana
     spec:
-      params:
-      ...
-      - name: infra-deployment-update-script
-        value: |
-         sed -i -e 's|\(https://github.com/redhat-appstudio/service-provider-integration-operator/config/monitoring/base?ref=\)\(.*\)|\1{{ revision }}|' components/monitoring/grafana/base/spi/kustomization.yaml
-        pipelineRef:
-         name: docker-build
-         bundle: quay.io/redhat-appstudio/hacbs-core-service-templates-bundle:latest
-      ...
+      configMapRef:
+        name: grafana-dashboard-o11y
+        key: o11y-dashboard.json
     ```
+ - Push the code to the team's repository
 
-  - Add a [volume](https://github.com/redhat-appstudio/infra-deployments/blob/2a6e4dcb272fa01d330b393d87b8f5ea5c434687/components/monitoring/grafana/base/grafana-app.yaml#L175) to `grafana-app.yaml` 
-    ```yaml
-    - name: grafana-dashboard-dora-metrics-volume
-      projected:
-       sources:
-         - configMap:
-             name: grafana-dashboard-dora-metrics
-     ```
-
-  - Add a [volumeMounts](https://github.com/redhat-appstudio/infra-deployments/blob/2a6e4dcb272fa01d330b393d87b8f5ea5c434687/components/monitoring/grafana/base/grafana-app.yaml#L87) to `grafana-app.yaml`
-    ```yaml
-    volumeMounts:
-    - mountPath: /var/lib/grafana/dashboards-dora-metrics
-      name: grafana-dashboard-dora-metrics-volume
-    ```
+ - Check out [this example o11y PR](https://github.com/redhat-appstudio/o11y/pull/39) 
+ for creating a dashboard in the team's repository
+  
 
-  - Add a link to the new maps folder in `appstudio-workload-monitoring` [ConfigMap](https://github.com/redhat-appstudio/infra-deployments/blob/2a6e4dcb272fa01d330b393d87b8f5ea5c434687/components/monitoring/grafana/base/grafana-app.yaml#L234) in  `grafana-app.yaml`
-    ```yaml
-    apiVersion: v1
-    kind: ConfigMap
-    metadata:
-      namespace: appstudio-workload-monitoring
-      name: grafana-dashboards
-    data:
-      default.yaml: |
-        apiVersion: 1
-        providers:
-          - name: Dora Metrics
-            folder: QE
-            type: file
-            disableDeletion: true
-            options:
-              path: /var/lib/grafana/dashboards-dora-metrics
-    ```
+#### infra-deployments repository
+Follow the next steps to add a dashborad to the `infra-deployments` repository
+
+  - Create a team folder under `components/monitoring/grafana/base/<team_name>`
```

Comment:
As those steps should be done one-after-the-other, use digits instead of dashes

Reply:
Updated to digits